Model/Quote/Checkout.php

```diff
@@ -184,32 +187,31 @@ private function getDeliveryData(): array
             $canHavePickup        = $consignment->canHaveDeliveryType(AbstractConsignment::DELIVERY_TYPE_PICKUP_NAME);
 
             $basePrice        = $this->helper->getBasePrice();
+            $mondayFee        = $canHaveMonday ? $this->helper->getMethodPrice($carrierPath, 'delivery/monday_fee') : 0;
             $morningFee       = $canHaveMorning ? $this->helper->getMethodPrice($carrierPath, 'morning/fee') : 0;
             $eveningFee       = $canHaveEvening ? $this->helper->getMethodPrice($carrierPath, 'evening/fee') : 0;
             $sameDayFee       = $canHaveSameDay ? (int) $this->helper->getMethodPrice($carrierPath, 'delivery/same_day_delivery_fee') : 0;
             $signatureFee     = $canHaveSignature ? $this->helper->getMethodPrice($carrierPath, 'delivery/signature_fee', false) : 0;
             $onlyRecipientFee = $canHaveOnlyRecipient ? $this->helper->getMethodPrice($carrierPath, 'delivery/only_recipient_fee', false) : 0;
             $isAgeCheckActive = $canHaveAgeCheck && $this->isAgeCheckActive($carrierPath);
 
-            $myParcelConfig['carrierSettings'][$carrier] = array_merge([
-                'allowDeliveryOptions'  => ! $this->package->deliveryOptionsDisabled && $this->helper->getBoolConfig($carrierPath, 'delivery/active'),
+            $myParcelConfig['carrierSettings'][$carrier] = [
+                'allowDeliveryOptions'  => ! $this->package->deliveryOptionsDisabled,
```

Comment:
Dit klopt wss niet, dit moet uit zijn als je geen pickup hebt. Ik denk dat je hier dezelfde waarde nodig hebt als bij `allowStandardDelivery`

Het verschil hier is dat het nu mogelijk maakt dat je standard delivery uitzet en bijvoorbeeld alleen ochtend en/of avondlevering aanbiedt. Ik zie dat dit niet duidelijk genoeg is in de upgrade docs dus dat zal ik aanpassen

Reply:
Ah bedankt voor je verheldering, ik denk dat ik ’m nu goed heb.